pkg/controllers/follower/controller.go

```diff
@@ -60,28 +59,13 @@ const (
 	EventReasonFailedUpdateFollower = "FailedUpdateFollower"
 )
 
-var (
-	// Map from supported leader type to pod spec path.
-	// TODO: think about whether PodTemplatePath/PodSpecPath should be specified in the FTC instead.
-	// Specifying in the FTC allows changing the path according to the api version.
-	// Other controllers should consider using the specified paths instead of hardcoded paths.
-	leaderPodSpecPaths = map[schema.GroupKind]string{
```

Comment:
We should continue to define an explicit list of `leaderPodSpecPaths` in this package as the source of truth for the supported leader types (although we can reuse the paths in `podutil`. If not an irrelevant change to the `podutil` might break the follower controller.